app/controllers/account_users_controller.rb

```diff
@@ -47,6 +48,7 @@ def destroy
     @account_user.deleted_by = session_user.id
 
     if @account_user.save
```

Comment:
Not the fault of the new code, but the new code sheds light on the existing code. Might be for another time, but the `if save` right next to a logging `delete` seems counter-intuitive. I wonder if it would be better to create a custom `destroy` method on `AccountUser`.